mailbox/api/src/main/java/org/apache/james/mailbox/events/EventDeadLettersHealthCheck.java

```diff
@@ -49,12 +49,13 @@ public Result check() {
             boolean containEvents = eventDeadLetters.containEvents().block();
 
             if (containEvents) {
+                LOGGER.warn("EventDeadLetters is degraded");
```

Comment:
EventDeadLetters is not empty. This means processing some mailbox events did fail. You might want to reDeliver these events.